config/locales/en.yml

```diff
@@ -2046,6 +2046,9 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   not_visible: not visible
   you_have_no_orders_yet: "You have no orders yet"
   show_only_complete_orders: "Only show complete orders"
+  successfully_created: ! '%{resource} has been successfully created!'
```

Comment:
What are those exclamation marks doing there? I found [some hint](https://stackoverflow.com/a/9703472/3377535) that this makes it a "vanilla string", but I have no idea why it wouldn't be a vanilla string.

Reply:
I copied these directly from Spree's translations. I'm not sure about the bang!

Reply:
If that's what they do in Spree, I'd just keep it. I agree that it's weird. Shall we merge?